.github/workflows/rust.yaml

```diff
@@ -54,15 +54,15 @@ jobs:
         run: ./contrib/test.sh
 
   MSRV:
-    name: Test - 1.56.1 toolchain
+    name: Test - 1.63.0 toolchain
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
     steps:
       - name: Checkout Crate
         uses: actions/checkout@v3
       - name: Checkout Toolchain
-        uses: dtolnay/rust-toolchain@1.56.1
+        uses: dtolnay/rust-toolchain@1.63.0
```

Comment:
I'm not going to hold this because the PR is blocking a bunch of other work but this should be read from `Cargo.toml` in a previous step.

Reply:
Holla if you can think of a better way of tracking issues across "core crates": https://github.com/rust-bitcoin/rust-bitcoin/issues/3335

Reply:
There are org-wide "projects" but I'm not sure if it's the right tool.